cognite_toolkit/_cdf_tk/loaders/_resource_loaders/extraction_pipeline_loaders.py

```diff
@@ -27,7 +27,6 @@
 )
 from cognite.client.data_classes.capabilities import (
     Capability,
-    ExtractionConfigsAcl,
```

Comment:
Pending, asking the extractor team whether this is used (or deprecated) as it is no longer in the API docs:
<img width="1049" height="915" alt="image" src="https://github.com/user-attachments/assets/6a69e8ad-2a80-41a6-a2dc-69fca7f2b740" />